dist/gmusic.js

```diff
@@ -207,6 +248,15 @@ proto.playback = {
       return GMusic.Playback.NO_SHUFFLE;
     }
   },
+  setShuffle: function (mode) {
+    if (mode === GMusic.Playback.NO_SHUFFLE || mode === GMusic.Playback.ALL_SHUFFLE) {
```

Comment:
While we can emulate an ENUM here, let's define one as a constant so it can be reused in other locations. The ideal location would be after `GMusic.Playback`:

```js
GMusic.Playback = { ... };
GMusic.Playback.SHUFFLE_MODES = [GMusic.Playback.NO_SHUFFLE, GMusic.Playback.ALL_SHUFFLE];
```

Then we can use an `indexOf` here:

```
if (GMusic.Playback.SHUFFLE_MODES.indexOf(mode) !== -1) {
  // while ...
}
```